### DISCUSSION TITLE
Adding Special Tags (Title, Description..)

### DISCUSSION THREAD

**Discussion by ahmed-esmail-1** [spatie/laravel-sitemap#533] [General]
Hello everyone, can I add special tags like title and description when I generate the Site map ?

            
            `<url>
        <loc>http://127.0.0.1:8000</loc>
        <lastmod>2022-01-01T00:00:00+00:00</lastmod>
        <changefreq>daily</changefreq>
        <priority>0.9</priority>
            </url>`

**Comment by dinfyru** [comment]
@ahmed-esmail-1 Did you find the answer?

**Reply by ahmed-esmail-1** [reply]
I did it manually

**Reply by dinfyru** [reply]
@ahmed-esmail-1 can you show example?)

**Reply by ahmed-esmail-1** [reply]
Of a sitemap or my code ?

**Reply by dinfyru** [reply]
final code with <title> tag

**Reply by ahmed-esmail-1** [reply]
For code I can't share it becasue it belongs to the company, but share your code with me and how you execute the command and I will modify it for you
